x-pack/plugins/data_enhanced/public/search/sessions_mgmt/components/actions/extend_button.tsx

```diff
@@ -36,9 +43,12 @@ const ExtendConfirm = ({
     defaultMessage: 'Cancel',
   });
   const message = i18n.translate('xpack.data.mgmt.searchSessions.extendModal.extendMessage', {
-    defaultMessage: "When would you like the search session '{name}' to expire?",
+    defaultMessage:
+      "The search session '{name}' expiration would be extended by {expiration}, until {newExpires}.",
     values: {
       name,
+      expiration: extendByDuration.humanize(),
```

Comment:
Nit: I don't think we need to display the duration in this dialog, and it might cause problems since we're not translating the `humanize` portion. If it just says "will be extended until {newExpires}", I don't think that's any less helpful.